### PR TITLE
Reviewer - EM  - Cacti Graphs config

### PR DESCRIPTION
**Pull request by johadalin**
This updates the cacti_host_template_sprout.xml file, to include a chronos all timers graph displaying the average, variance, and HWM,LWM for the chronos global counts table. 
This is a direct export from the working cacti node, and has been tested uploaded onto another node, producing a working graph, so i would think it shouldn't cause any problems. 

I don't know when it would want to be merged, as if the allTimersTable code is not running on a deployment but this file is, cacti will at least dhow a blank graph, but may even show errors on looking for graph data that does not exist.

### CODE REVIEW

File: cookbooks/clearwater/files/default/cacti/templates/cacti_host_template_sprout.xml [hunk 1]
```diff
@@ -1146,7 +1319,7 @@
 				<t_data_source_type_id></t_data_source_type_id>
 				<data_source_type_id>2</data_source_type_id>
 				<t_rrd_heartbeat></t_rrd_heartbeat>
-				<rrd_heartbeat>600</rrd_heartbeat>
+				<rrd_heartbeat>120</rrd_heartbeat>
```
Comment (eleanor-merry) [inline cookbooks/clearwater/files/default/cacti/templates/cacti_host_template_sprout.xml]: Why have these values changed?

File: cookbooks/clearwater/files/default/cacti/templates/cacti_host_template_sprout.xml [hunk 2]
```diff
@@ -2807,38 +3308,6 @@
 		<timespan>14400</timespan>
 		<cf_items>1|3</cf_items>
 	</hash_150024283ea2bf1634d92ce081ec82a634f513>
-	<hash_150024c21df5178e5c955013591239eb0afd46>
```
Comment (eleanor-merry) [inline cookbooks/clearwater/files/default/cacti/templates/cacti_host_template_sprout.xml]: Why has this been deleted?

**Comment by eleanor-merry** [comment]
The Chronos changes look good, but it looks like there's a number of unrelated changes. Can you look through these and either let me know why they're needed or remove them?

**Comment by johadalin** [comment]
Closing PR as it is superseded by PR #249

**Pull request closed without merging**